src/App.tsx

```diff
@@ -1,10 +1,40 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Error } from './components/ErrorMessage';
+import { Footer } from './components/Footer';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
 
 export const App: React.FC = () => {
+  const  [ todos ,  setTodos ]  =  useState < Todo [ ] > ( [ ] ) ;
+  const  [ errorMessage ,  setErrorMessage ]  =  useState ( '' ) ;
+  const  [ status ,  setStatus ]  =  useState ( Status . All ) ;
```

Comment:
Don't forget about code styles
```suggestion
  const  [todos, setTodos] = useState<Todo[]>([]);
  const  [errorMessage, setErrorMessage] = useState('') ;
  const  [status,  setStatus] = useState(Status.All) ;
```